src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -246,79 +139,42 @@ public Network cloneNetworkVariant(UUID networkUuid, String originVariantId, Str
     }
 
     @Transactional(readOnly = true)
-    public List<ModificationInfos> applyModifications(Network network, UUID networkUuid, BuildInfos buildInfos) {
-        NetworkStoreListener listener = NetworkStoreListener.create(network, networkUuid, null, networkModificationRepository, equipmentInfosService, true, true);
-
+    public List<ModificationInfos> buildVariant(@NonNull NetworkInfos networkInfos, @NonNull BuildInfos buildInfos) {
```

Comment:
It's not easy to understand this function, at least for me, maybe some comments ?